src/js/lib/logging/test/test.js

```diff
@@ -0,0 +1,79 @@
+const assert = require('assert');
```

Comment:
Our CI config is at `.github/workflows/check.yml`. I was never creating workflows for JS tests, so I cant help more with it now. In case you'd not be able to do it fast, lets leave it without this connection and lets do it as a separate pr some day in the future.

Reply:
To me it seems like this should better be included when you execute `./run test`, which the CI currently does. Am I right?

Reply:
Oh, you are right. This way it would be much better. Thanks for catching it, I was definitely tired writing it ;)

Reply:
Then it should better be implemented by someone who knows how the run script works. You just need to execute `npm test` on the log-server package.

Reply:
Sure, we will implement it in separate PR then!